src/modules/proto_control/radmin.c

```diff
@@ -838,6 +838,7 @@ int main(int argc, char **argv)
 
 		uid = getuid();
 		gid = getgid();
+		subcs = NULL;
```

Comment:
This shouldn't be needed.

Reply:
@alandekok It was to mute the clang.

```
src/modules/proto_control/radmin.c:846:12: warning: 2nd function call argument
      is an uninitialized value
  ...subcs = cf_section_find_next(cs, subcs, "server", server);
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/modules/proto_control/radmin.c:858:20: warning: 2nd function call argument
      is an uninitialized value
  ...((subcs = cf_section_find_next(cs, subcs, "server", CF_IDENT_ANY)) != NU...
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2 warnings generated.
Decode with:```

Reply:
https://travis-ci.org/FreeRADIUS/freeradius-server/jobs/549750054